### PR TITLE
Upload site

### PR DESCRIPTION
**Pull request by lieuwex**
This still needs to be able to be opened with a click on the dropzone.

### CODE REVIEW

**Comment by lieuwex** [comment]
Maybe a loading indicator would be nice too

File: site/index.html [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+		var arrbuf = reader.result;
+
+		var xhr = new XMLHttpRequest();
+
+		xhr.onreadystatechange = function (e) {
+			if (xhr.readyState !== 4) {
+				return;
+			}
+
+			if (xhr.status === 200) {
```
Comment (lieuwex) [inline site/index.html]: Oh wait now I realise I didn't even check if the server returns 200 or something like a created status code.

**Comment by tomsmeding** [comment]
Server most probably returns 200. An xhr has an onprogress property that you can install a handler on, which should bw nice for progress reporting.

**Comment by lieuwex** [comment]
Do we really want a nice progress bar, or just something like 'uploading...'?

**Comment by tomsmeding** [comment]
What might be nice is a percentage counter, like
Uploading... (42%)
Where the percentage changes as the upload progresses. Possibly also a total size? Or maybe (312K/4M) instead?

**Comment by lieuwex** [comment]
I will create a PR later for click to upload. :)

**Comment by tomsmeding** [comment]
Looks nice, will upload in a moment, hang on

**Pull request merged**